### PR TITLE
fix(readme): update milestones

### PR DESCRIPTION
**Pull request by rubiin**
just a small fix, as we seem to be hardcoding the shell names

### CODE REVIEW

**Comment by rubiin** [comment]
also https://github.com/kRHYME7/Hyde-cli#roadmap , I might be able to help on some . Do make issues so I would know what to do for each

File: Scripts/FUNCTIONS-hyde [hunk 1]
```diff
@@ -192,7 +192,7 @@ rename_backup() {
 shell_select() {
     #TODO for now the original script requires deletion of 1 of the shell to change shell therefore I should create a different approach
     navigate_clone
-    myShell="$(echo -e "zsh\nfish" | Fzf-Menu "${FUNCNAME[0]}")"
+    myShell="$(chsh -l | awk '/\/usr\/bin\/zsh/ || /\/usr\/bin\/fish/' | sed 's/\/usr\/bin\///' | Fzf-Menu "${FUNCNAME[0]}")"
```
Comment (kRHYME7) [inline Scripts/FUNCTIONS-hyde:195]: How about if 1 of the shells is not yet installed?

**Comment by kRHYME7** [comment]
Here yah go.

I'm almost finished working on a draft for package installation.

**Comment by rubiin** [comment]
done

**Review by kRHYME7 — CHANGES_REQUESTED** [review]
Thanks 
.  

But please fix the shell_select

File: Scripts/FUNCTIONS-hyde (hunk 1 above)
Comment (rubiin) [reply Scripts/FUNCTIONS-hyde:195]: this assumes if one of them is available. Either fish or zsh . The chances of one of them not being installed is pretty low imo

**Comment by kRHYME7** [comment]
Thank you 🎇

**Pull request merged**